Patch developed inhouse and fixes a Solaris specific issue related to user creation
in snmpv3. The patch has been submitted to the community but has
not been accepted yet.

The details can be found in the following location
https://sourceforge.net/p/net-snmp/bugs/2689/

--- net-snmp-5.4.1/net-snmp-config.in	2015-11-04 02:33:59.029951910 -0800
+++ copy_net-snmp-5.4.1/net-snmp-config.in	2015-11-04 02:36:34.650208866 -0800
@@ -191,7 +191,7 @@
       ;;
     ####################################################
     --create-snmpv3-user)
-      if " $PSPROG -z `zonename` | egrep ' snmpd *$' " > /dev/null 2>&1 ; then
+      if /usr/bin/ps -z `zonename` | egrep ' snmpd *$' > /dev/null 2>&1 ; then
          echo "Apparently at least one snmpd demon is already running."
          echo "You must stop them in order to use this command."
          exit 1
